### DISCUSSION TITLE
How to install horovod with old version TF2.2

### DISCUSSION THREAD

**Discussion by ShrimpLau** [horovod/horovod#3334] [Q&A]
I'm trying to install horovod with `HOROVOD_GPU_OPERATIONS=NCCL HOROVOD_WITH_TENSORFLOW=1 pip install horovod[tensorflow,keras]`, but it must uninstall my current TF2.2 and install the latest tensorflow 2.6, how can I fix it? 
I have tried install horovod==1.19.0 but it does not help. 
Look forward to any reply

**Comment by maxhgerlach** [comment]
If you have already installed TensorFlow and Keras before, it should be fine to remove the `[tensorflow,keras]` extra requirement specification.

Something like the following should work and the Horovod install should not change the installed version of TF (it's the same with Keras, I'm just not sure personally about versioning there):
```
pip install -U tensorflow==2.2
HOROVOD_GPU_OPERATIONS=NCCL HOROVOD_WITH_TENSORFLOW=1 pip install horovod
```

**Reply by ShrimpLau** [reply]
I have successfully installed it by compiling the source code, anyway, thanks for your reply!
